templates/MailPreview/email.php

```diff
@@ -26,8 +26,8 @@
 
 <script>
     function formatChanged(form) {
-        var part_name = form.options[form.selectedIndex].value
-        var iframe = document.getElementsByName('messageBody')[0];
+        let part_name = form.options[form.selectedIndex].value
+        let iframe = document.getElementsByName('messageBody')[0];
         iframe.contentWindow.location.replace('?part=' + part_name + '&plugin=<?= h($plugin); ?>');
```

Comment:
We might be able to move this into an event handler that reads from global variables, or variables that are set into the toolbar script as script tag attributes. :shrug:

Reply:
Do we really need to introduce another JS file just for those 4 Lines of JS?
This is only needed on the https://myurl/debug-kit/mail-preview/preview/MyMailPreview/someAction?plugin= template which only has a very minimal amount of CSS and JS.

Reply:
That's fair Lets leave this here then.